jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/Checkbox.java

```diff
@@ -21,34 +31,29 @@ public boolean isNotChecked() {
     }
 
     @JDIAction("Check '{name}'")
-    @Override
     public void check() {
         if (isNotChecked()) {
-            click();
+            getCheckBoxes().click();
         }
     }
 
     @JDIAction("Uncheck '{name}'")
-    @Override
     public void uncheck() {
         if(isChecked()) {
-            click();
+            getCheckBoxes().click();
         }
     }
 
     @JDIAction("Is '{name}' disabled")
-    @Override
```

Comment:
Please don't delete Override. The same for other methods where use can override

Reply:
Fixed it